default/stringtables/en.txt

```diff
@@ -6197,6 +6202,18 @@ SP_EXOBOT_DESC
 '''While not as efficient as fully self-directed citizens, Exobots can colonize otherwise inhospitable planets, and build more Exobots.
 '''
 
+SP_ANCIENT_GUARDIANS
+Ancient Guardians
+SP_ANCIENT_GUARDIANS_GAMEPLAY_DESC
+'''Guardian Robots constructed by the Precursor civilizations aeons ago
```

Comment:
Better to say "a Precursor", we've got several in game already, no need to say which one.